views/pages/DashboardPage/DashboardPageSectionHeader.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Typography } from '@mui/material'
+
+import { fontSize } from '../../../constants'
+
+const DashboardPageSectionHeader = ({ title, to }) => {
```

Comment:
Should this be in `components` with its own test? I could go either way, and I like splitting this out, but in theory it could also go in the page itself right? It isn't going to be used elsewhere?

Reply:
Ooooh I see it's used twice within the page.. maybe this should just be a SectionHeader generic component?